website/docs/d/project.html.markdown

```diff
@@ -86,13 +86,9 @@ In addition to all arguments above, the following attributes are exported:
 * `org_id` - The ID of the organization you want to create the project within.
 * `cluster_count` - The number of Atlas clusters deployed in the project.
 * `created` - The ISO-8601-formatted timestamp of when Atlas created the project.
-* `teams.#.team_id` - The unique identifier of the team you want to associate with the project. The team and project must share the same parent organization.
-* `teams.#.role_names` - Each string in the array represents a project role assigned to the team. Every user associated with the team inherits these roles. The [MongoDB Documentation](https://www.mongodb.com/docs/atlas/reference/user-roles/#organization-roles) describes the roles a user can have.
-* `limits.#.name` - Human-readable label that identifies this project limit.
-* `limits.#.value` - Amount the limit is set to.
-* `limits.#.current_usage` - Amount that indicates the current usage of the limit.
-* `limits.#.default_limit` - Default value of the limit.
-* `limits.#.maximum_limit` - Maximum value of the limit.
+* `teams` - Returns all teams to which the authenticated user has access in the project. See [Teams](#teams).
+* `limits` - The limits for the specified project. See [Limits](#limits).
+* `ip_addresses` - IP addresses in a project categorized by services. See [IP Addresses](#ip-addresses).
```

Comment:
Adjusting format of defining nested schema in other section, generates consistency with project resource documentation and how tfplugindocs also defines nested schemas.

Reply:
nice!